public/css/signUp.css

```diff
@@ -68,93 +68,50 @@ body {
     background: #dfe7e9;
     font-family: 'Roboto', sans-serif;
 }
-.form-control {
-    font-size: 16px;
-    transition: all 0.4s;
-    box-shadow: none;
-    background-color: #f1f1f1;
-}
-.form-control:focus {
-    border-color: #5cb85c;
-}
-.form-control, .btn {
-    border-radius: 50px;
-    outline: none !important;
-}
-
 .signup-form {
     position: relative;
+    height: 100vh;
     top: -25px;
```

Comment:
Negative values are not always a good option. Try to fix it some other way. This will break in some browser or the other